utils/frame/remote-externalities/Cargo.toml

```diff
@@ -13,22 +13,22 @@ readme = "README.md"
 targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
-jsonrpsee-ws-client = { version = "=0.2.0-alpha.6", default-features = false }
-jsonrpsee-proc-macros = "=0.2.0-alpha.6"
+jsonrpsee-ws-client = { version = "0.2.0", default-features = false, features = ["tokio02"] }
+jsonrpsee-proc-macros = "0.2.0"
 
 hex = "0.4.0"
 env_logger = "0.8.2"
 log = "0.4.11"
 codec = { package = "parity-scale-codec", version = "2.0.0" }
-
 serde_json = "1.0"
+serde = "1.0.0"
 
 sp-io = { version = "3.0.0", path = "../../../primitives/io" }
 sp-core = { version = "3.0.0", path = "../../../primitives/core" }
 sp-runtime = { version = "3.0.0", path = "../../../primitives/runtime" }
 
 [dev-dependencies]
-tokio = { version = "1.6.0", features = ["macros", "rt"] }
```

Comment:
Why downgrade from 1.6 to 0.2?

Reply:
For compatibility with `jsonrpsee` v0.2.0